### ISSUE TITLE
error on listing episodes of Kulderzipken

### ISSUE THREAD

**Issue by dwalerXIII** [add-ons/plugin.video.vrt.nu#828]
### Describe the bug ###

listing of Kulderzipke gives error while others work well.
On https://www.vrt.be/vrtnu/a-z/kulderzipken/1/kulderzipken-s1a1/ the episodes are available.

### To Reproduce ###
1. Go to Kanalen
2. Click on Ketnet
3. Scroll down to Kulderzipken

### Expected behavior ###

no list of episodes is shown
Episodes of other shows can be shown

### Additional context ###

- Operating system: OSMC on raspberry pi3
- Kodi version: 18.8
- Addon version: 2.4.0
- Using a VPN: no
- Country you are using the addon from: Belgium

### Log (if available) ###
```paste below

2020-10-31 21:39:12.796 T:1300230368   DEBUG: StartScript - calling plugin VRT NU('plugin://plugin.video.vrt.nu/programs/kulderzipken','29','','resume:false')
2020-10-31 21:39:12.797 T:1237307616   DEBUG: Thread LanguageInvoker start, auto delete: false
2020-10-31 21:39:12.797 T:1237307616    INFO: initializing python engine.
2020-10-31 21:39:12.797 T:1237307616   DEBUG: CPythonInvoker(35, /home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon_entry.py): start processing
2020-10-31 21:39:13.068 T:1237307616   DEBUG: -->Python Interpreter Initialized<--
2020-10-31 21:39:13.069 T:1237307616   DEBUG: CPythonInvoker(35, /home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon_entry.py): the source file to load is "/home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon_entry.py"
2020-10-31 21:39:13.069 T:1237307616   DEBUG: CPythonInvoker(35, /home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon_entry.py): setting the Python path to /home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib:/home/osmc/.kodi/addons/script.module.beautifulsoup4/lib:/home/osmc/.kodi/addons/script.module.dateutil/lib:/home/osmc/.kodi/addons/script.module.inputstreamhelper/lib:/home/osmc/.kodi/addons/script.module.routing/lib:/home/osmc/.kodi/addons/script.module.six/lib:/usr/lib/python2.7:/usr/lib/python2.7/plat-arm-linux-gnueabihf:/usr/lib/python2.7/lib-tk:/usr/lib/python2.7/lib-old:/usr/lib/python2.7/lib-dynload:/usr/local/lib/python2.7/dist-packages:/usr/lib/python2.7/dist-packages:/usr/lib/python2.7/dist-packages/PILcompat:/usr/lib/python2.7/dist-packages/gtk-2.0
2020-10-31 21:39:13.069 T:1237307616   DEBUG: CPythonInvoker(35, /home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon_entry.py): entering source directory /home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib
2020-10-31 21:39:13.070 T:1237307616   DEBUG: CPythonInvoker(35, /home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon_entry.py): instantiating addon using automatically obtained id of "plugin.video.vrt.nu" dependent on version 2.25.0 of the xbmc.python api
2020-10-31 21:39:13.519 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: loading setting definitions
2020-10-31 21:39:13.519 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: trying to load setting definitions from old format...
2020-10-31 21:39:13.522 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: loading setting values
2020-10-31 21:39:13.522 T:1237307616   DEBUG: CSettingsManager: requested setting (adaptive_settings) was not found.
2020-10-31 21:39:13.522 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: failed to find definition for setting adaptive_settings. Creating a setting on-the-fly...
2020-10-31 21:39:13.522 T:1237307616   DEBUG: CSettingsManager: requested setting (delete_tokens) was not found.
2020-10-31 21:39:13.522 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: failed to find definition for setting delete_tokens. Creating a setting on-the-fly...
2020-10-31 21:39:13.523 T:1237307616   DEBUG: CSettingsManager: requested setting (invalidate_caches) was not found.
2020-10-31 21:39:13.523 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: failed to find definition for setting invalidate_caches. Creating a setting on-the-fly...
2020-10-31 21:39:13.523 T:1237307616   DEBUG: CSettingsManager: requested setting (refresh_favorites) was not found.
2020-10-31 21:39:13.523 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: failed to find definition for setting refresh_favorites. Creating a setting on-the-fly...
2020-10-31 21:39:13.523 T:1237307616   DEBUG: CSettingsManager: requested setting (usetwitter) was not found.
2020-10-31 21:39:13.523 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: failed to find definition for setting usetwitter. Creating a setting on-the-fly...
2020-10-31 21:39:13.523 T:1237307616   DEBUG: CSettingsManager: requested setting (widevine_install) was not found.
2020-10-31 21:39:13.523 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: failed to find definition for setting widevine_install. Creating a setting on-the-fly...
2020-10-31 21:39:13.657 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: loading setting definitions
2020-10-31 21:39:13.657 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: trying to load setting definitions from old format...
2020-10-31 21:39:13.660 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: loading setting values
2020-10-31 21:39:13.661 T:1237307616   DEBUG: CSettingsManager: requested setting (adaptive_settings) was not found.
2020-10-31 21:39:13.661 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: failed to find definition for setting adaptive_settings. Creating a setting on-the-fly...
2020-10-31 21:39:13.661 T:1237307616   DEBUG: CSettingsManager: requested setting (delete_tokens) was not found.
2020-10-31 21:39:13.661 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: failed to find definition for setting delete_tokens. Creating a setting on-the-fly...
2020-10-31 21:39:13.661 T:1237307616   DEBUG: CSettingsManager: requested setting (invalidate_caches) was not found.
2020-10-31 21:39:13.661 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: failed to find definition for setting invalidate_caches. Creating a setting on-the-fly...
2020-10-31 21:39:13.661 T:1237307616   DEBUG: CSettingsManager: requested setting (refresh_favorites) was not found.
2020-10-31 21:39:13.661 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: failed to find definition for setting refresh_favorites. Creating a setting on-the-fly...
2020-10-31 21:39:13.661 T:1237307616   DEBUG: CSettingsManager: requested setting (usetwitter) was not found.
2020-10-31 21:39:13.661 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: failed to find definition for setting usetwitter. Creating a setting on-the-fly...
2020-10-31 21:39:13.661 T:1237307616   DEBUG: CSettingsManager: requested setting (widevine_install) was not found.
2020-10-31 21:39:13.661 T:1237307616   DEBUG: CAddonSettings[plugin.video.vrt.nu]: failed to find definition for setting widevine_install. Creating a setting on-the-fly...
2020-10-31 21:39:13.662 T:1237307616    INFO: [plugin.video.vrt.nu] Access: plugin://plugin.video.vrt.nu/programs/kulderzipken
2020-10-31 21:39:13.662 T:1237307616   DEBUG: [plugin.video.vrt.nu][routing] Dispatching to 'programs', args: {u'program': u'kulderzipken'}
2020-10-31 21:39:13.670 T:1237307616  NOTICE: [plugin.video.vrt.nu] Got item from cache '/home/osmc/.kodi/userdata/addon_data/plugin.video.vrt.nu/cache/favorites.json'
2020-10-31 21:39:13.677 T:1237307616  NOTICE: [plugin.video.vrt.nu] Got item from cache '/home/osmc/.kodi/userdata/addon_data/plugin.video.vrt.nu/cache/resume_points.json'
2020-10-31 21:39:13.681 T:1237307616  NOTICE: [plugin.video.vrt.nu] URL get: https://vrtnu-api.vrt.be/search?i=video&facets[programUrl]=//www.vrt.be/vrtnu/a-z/kulderzipken/&size=300
2020-10-31 21:39:14.221 T:1237307616   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.AttributeError'>
                                            Error Contents: 'NoneType' object has no attribute 'get'
                                            Traceback (most recent call last):
                                              File "/home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon_entry.py", line 14, in <module>
                                                run(argv)
                                              File "/home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon.py", line 360, in run
                                                plugin.run(argv)
                                              File "/home/osmc/.kodi/addons/script.module.routing/lib/routing.py", line 130, in run
                                                self._dispatch(self.path)
                                              File "/home/osmc/.kodi/addons/script.module.routing/lib/routing.py", line 141, in _dispatch
                                                view_func(**kwargs)
                                              File "/home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon.py", line 164, in programs
                                                VRTPlayer().show_episodes_menu(program=program, season=season)
                                              File "/home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib/vrtplayer.py", line 255, in show_episodes_menu
                                                episode_items, sort, ascending, content = self._apihelper.list_episodes(program=program, season=season)
                                              File "/home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib/apihelper.py", line 118, in list_episodes
                                                return self.__map_seasons(program, seasons, episodes)
                                              File "/home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib/apihelper.py", line 179, in __map_seasons
                                                tag=self._metadata.get_tag(episode)),
                                              File "/home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib/metadata.py", line 766, in get_tag
                                                for category in api_data.get('categories')])
                                            AttributeError: 'NoneType' object has no attribute 'get'
                                            -->End of Python script error report<--
2020-10-31 21:39:14.221 T:1237307616   DEBUG: onExecutionDone(35, /home/osmc/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon_entry.py)
2020-10-31 21:39:14.229 T:1916191664   DEBUG: ------ Window Init (DialogNotification.xml) ------
2020-10-31 21:39:14.229 T:1864360160    INFO: CActiveAESink::OpenSink - initialize sink
2020-10-31 21:39:14.229 T:1864360160   DEBUG: CActiveAESink::OpenSink - trying to open device PI:HDMI
2020-10-31 21:39:14.229 T:1864360160   DEBUG: CAESinkPi:Initialize Format:15 Channels:2 Samplerate:44100 framesize:8 bufsize:17640 bytes/s=352800.00 dest=PI:HDMI
2020-10-31 21:39:14.230 T:1864360160   DEBUG: CAESinkPi:SetAudioProps hdmi_stream_channels 0 hdmi_channel_map 00000008
2020-10-31 21:39:14.231 T:1864360160   DEBUG: COMXCoreComponent::Initialize OMX.broadcom.audio_render input port 100 output port 100 m_handle 0x6de02e60
2020-10-31 21:39:14.233 T:1864360160   DEBUG: COMXCoreComponent::AllocInputBuffers component(OMX.broadcom.audio_render) - port(100), nBufferCountMin(1), nBufferCountActual(2), nBufferSize(17648), nBufferAlignment(16)
2020-10-31 21:39:14.233 T:1864360160   DEBUG: CActiveAESink::OpenSink - SinkPi Initialized:
2020-10-31 21:39:14.233 T:1864360160   DEBUG:   Output Device : HDMI
2020-10-31 21:39:14.233 T:1864360160   DEBUG:   Sample Rate   : 44100
2020-10-31 21:39:14.233 T:1864360160   DEBUG:   Sample Format : AE_FMT_FLOAT
2020-10-31 21:39:14.233 T:1864360160   DEBUG:   Channel Count : 2
2020-10-31 21:39:14.233 T:1864360160   DEBUG:   Channel Layout: FL, FR
2020-10-31 21:39:14.233 T:1864360160   DEBUG:   Frames        : 2205
2020-10-31 21:39:14.233 T:1864360160   DEBUG:   Frame Size    : 8
2020-10-31 21:39:14.394 T:1237307616    INFO: Python interpreter stopped
2020-10-31 21:39:14.394 T:1237307616   DEBUG: Thread LanguageInvoker 1237307616 terminating
2020-10-31 21:39:14.405 T:1300230368   ERROR: GetDirectory - Error getting plugin://plugin.video.vrt.nu/programs/kulderzipken

```

**Comment by dagwieers** [comment]
VRT NU has added a new category named "Nostalgie" and because our add-on did not know about this category it fails. We discovered this yesterday and fixed it, but the fix has not yet been distributed to Kodi. This may take 24 hours when accepted.
- Leia: https://github.com/xbmc/repo-plugins/pull/3187
- Matrix: https://github.com/xbmc/repo-plugins/pull/3188

**Comment by dagwieers** [comment]
Fixed in #825
